### PR TITLE
[docs] Added requested information to Interactive tooltip usage page

### PR DESCRIPTION
**Pull request by connor-leech**
In response to https://github.com/carbon-design-system/carbon-website/issues/1492.

* Adjusted header weights and standardizing pattern to allow for expanded guidance.
* Added additional guidance for interactive tooltip.

It still looks weird with only a line of copy beneath "general guidance" but that's a pattern that persists across our usage pages. More content will follow as part of an upcoming component content push.

Opened issue for contextual images on all of our component pages (https://github.com/carbon-design-system/carbon-website/issues/1587). Currently, too many images just float in space without context.

### CODE REVIEW

**Comment by dakahn** [comment]
@carbon-design-system/design 👋

File: src/content/components/tooltip/usage.mdx [hunk 1]
```diff
@@ -4,34 +4,73 @@ tabs: ['Code', 'Usage','Style']
...
 ## General guidance
 
-Tooltips provide additional information upon hover or focus. They often contain helper text that is contextual to an element.
+Tooltips provide additional information upon hover or focus.
 
-#### Icon tooltip
+## Variants
 
-An icon tooltip is used to clarify the action or name of an interactive icon button. The tooltip content should only contain one to two words. Icon tooltips appear on `hover` and `focus`.
+A tooltip is to provide additional, contextual information. Each variant achieves this for different design needs.
```
Comment (aagonzales) [inline src/content/components/tooltip/usage.mdx]: Seems a little redundant with the general guidance paragraph.

File: src/content/components/tooltip/usage.mdx [hunk 2]
```diff
@@ -4,34 +4,73 @@ tabs: ['Code', 'Usage','Style']
 ---
 ## General guidance
```
Comment (aagonzales) [inline src/content/components/tooltip/usage.mdx]: I think you should just kill this section. It serves no real purpose.

Comment (connor-leech) [reply src/content/components/tooltip/usage.mdx]: I agree, this whole page needs work. Part of the usage overhaul!

**Pull request merged**